modules/sdk-core/src/bitgo/enterprise/enterprise.ts

```diff
@@ -148,4 +148,23 @@ export class Enterprise implements IEnterprise {
       challenge
     );
   }
+
+  /**
+   * Fetches the existing TSS ECDSA enterprise challenge if one exists.
+   * Can be used with uploadAndEnableTssEcdsaSigning to re-sign the
+   * enterprise challenge with new signatures.
+   */
+  async getExistingTssEcdsaChallenge(): Promise<NTilde> {
+    const urlPath = `/api/v2/enterprise/${this.id}/tssconfig`;
+    const tssConfig = await this.bitgo.get(this.bitgo.url(urlPath, 2)).send().result();
```

Comment:
Can we reuse bitgoapi's `getChallengesForEcdsaSigning` to do this request?

Reply:
No, thats a completely diff endpoint

Reply:
Given its only gonna be used for enterprise, i think its fine as is